src/v2/Apps/_Preferences2/PreferencesApp.tsx

```diff
@@ -1,9 +1,164 @@
-import { Text } from "@artsy/palette"
+import {
+  Button,
+  Checkbox,
+  Column,
+  GridColumns,
+  Separator,
+  Text,
+} from "@artsy/palette"
+import { Formik } from "formik"
 
 export const PreferencesApp: React.FC = () => {
   return (
     <>
-      <Text variant="xl">Preferences App</Text>
+      <Text variant="xl" my={4}>
+        Preferences Center
+      </Text>
+
+      <Separator my={4} />
+
+      <Formik
+        initialValues={{
+          first: false,
+          second: false,
+          third: false,
+          fourth: false,
+          fifth: false,
+        }}
+        onSubmit={values => {
+          console.log(values)
```

Comment:
(I'm assuming that a follow-up PR will handle this bit; this is just a lil bit of prework)

Reply:
Yup, hoping to work on this chunk once we successfully update MP.